templates/takeovers/index.html

```diff
@@ -110,4 +110,6 @@
 {% include "takeovers/_anbox-cloud-webinar.html" %}
 <p>26 Feb 2020</p>
 {% include "takeovers/_smart-start-webinar.html" %}
+<p>4th Mar 2020</p>
```

Comment:
It's the 2nd, or are we holding off publishing this till Weds?

Reply:
I guess so - unless we hear otherwise